### PR TITLE
Add eCMS Languages custom module

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

New custom module to provide configurable language switcher. Config form to select excluded languages, link_alter to modify the language switcher. Update hook to install new mod, uninstall the 'disable language' module.

UPDATE:
Realized we need to leave disable_language in place for the uninstall to complete successfully.
Uninstalls the scheduler module as well, and removes associated config and code from the workflow.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | yes
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | yes
| Risk level | Low
| Relevant links | JIRA issue, bug reports, security alerts, etc.

### CODE REVIEW

File: ecms_base/ecms_base.profile [hunk 1]
```diff
@@ -156,3 +156,25 @@ function ecms_base_update_9020(array &$sandbox): void {
   // Make sure necessary modules are installed.
   \Drupal::service('module_installer')->install($modules_to_install);
 }
+
+/**
+ * Updates to run for the 0.2.1 tag.
+ */
+function ecms_base_update_9022(array &$sandbox): void {
```
Comment (pfrilling) [inline ecms_base/ecms_base.profile:163]: Hook number should be `9021`.

File: ecms_base/modules/custom/ecms_languages/src/Form/EcmsLanguageSettings.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
+<?php
+
```
Comment (pfrilling) [inline ecms_base/modules/custom/ecms_languages/src/Form/EcmsLanguageSettings.php:2]: Add the strict type flag here.

File: ecms_base/modules/custom/ecms_languages/src/Form/EcmsLanguageSettings.php [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+   *
+   * @var \Drupal\Core\Language\LanguageManager
+   */
+  protected $languageManager;
+
+  /**
+   * Class constructor.
+   *
+   * @param \Drupal\Core\Config\ConfigFactoryInterface $config_factory
+   *   The plugin factory to create the condition plugin.
```
Comment (pfrilling) [inline ecms_base/modules/custom/ecms_languages/src/Form/EcmsLanguageSettings.php:27]: Nit: The comment doesn't match the variable. Maybe `the config.factory service.`. I personally always comment these with the actual service name, mostly so I remember the service name.

**Review by pfrilling — CHANGES_REQUESTED** [review]
Found a few minor changes. Not required, but you might want to add a schema file for your new configuration.

**Comment by ben-hamelin** [comment]
@pfrilling Ok I think I hit those. Needed to update the unit test as well so waiting to see if it passes now.

**Review by pfrilling — APPROVED** [review]
Looks good. 🚢

**Pull request merged**